next.config.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is there a reason for this change or did you add it accidentally to git? I prefer to keep the git log clean and only include relevant changes, because it makes git forensics (with `git log <file>` or `git blame` cleaner).

Reply:
ah yes, some mess from a previous change